Microsoft.Azure.CosmosRepository.Samples/ServiceTier/Program.cs

```diff
@@ -67,8 +66,10 @@ static async Task RawRepositoryExampleAsync(IRepository<Person> repository)
             _ = await repository.CreateAsync(new[] { maryShaw, calvinWeatherfield });
 
             // Reading...
-            Person mary = await repository.GetAsync(maryShaw.Id, maryShaw.SyntheticPartitionKey);
+            Person? mary = await repository.GetAsync(maryShaw.Id, maryShaw.SyntheticPartitionKey);
+            _ = mary ?? throw new NullReferenceException(nameof(mary));
```

Comment:
Why is this necessary?

Reply:
GetAsync can return a null so you need the reference nullable Person. The null check after that may just have been me fixing a warning on the next line (not shown), but it would blow up either way without it so I don't suppose it makes a difference.